src/main/java/com/example/sixthmafiabot/exceptions/ValidateException.java

```diff
@@ -0,0 +1,13 @@
+package com.example.sixthmafiabot.exceptions;
+
+import com.example.sixthmafiabot.exceptions.Abstract.BaseCustomException;
+
+import java.util.Map;
+
+public class ValidateException extends BaseCustomException {
+
+    public ValidateException(Map<String, String> exceptionMap) {
+
+        super(exceptionMap.toString());
```

Comment:
do not cast it to str
we will handle this exception as dictionary later

Reply:
+